src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
змінні краще робити якомого більш закритими якщо до них не потрібен доступ ззовні + якщо клас не immutable капс неймнг не потрібно використовувати 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names